tests/integration/security/sds_egress/main_test.go

```diff
@@ -39,6 +39,8 @@ func TestMain(m *testing.M) {
 		NewSuite("sds_egress_workload_mtls_istio_mutual_test", m).
 		Label(label.CustomSetup).
 		RequireEnvironment(environment.Kube).
+		// SDS requires Kubernetes 1.13
+		RequireEnvironmentVersion(13).
```

Comment:
@Monkeyanator I think your test works without SDS?

`ISTIO_MUTUAL` is a mode on gateway, in case of non-SDS is configured, gateway will use key/cert as file system path as credentials for its listeners.

Reply:
Yes, this test should work with or without SDS (despite it being under `sds_egress` tests).

Reply:
Never mind, I went back and checked the specific test logic: it makes sure that using TLS with key/cert paths fails, so it does expect SDS.

Reply:
I look at the tests that are actually failnig when we had the k8s 1.12 test and tihs was one. I think you cannot even pass `values-istio-sds-auth.yaml` in 1.12, so the setup fails